<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect_form</class>
 <widget class="QWidget" name="Connect_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to DB</string>
  </property>
  <widget class="QLabel" name="IP_address_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>21</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>IP address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP_address_textbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Login_texbox">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>70</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Login_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>45</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pass_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>78</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="DB_name_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DataBase name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DB_name_textbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pass_texxtbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect to DB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
